modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -136,21 +136,22 @@ function editProbandInfoFields($db, $user)
         header("HTTP/1.1 403 Forbidden");
         exit;
     }
-
-    $candID = $_POST['candID'];
+    //Sanitizing the post data
+    $sanitize = array_map('htmlentities', $_POST);
```

Comment:
@johnsaigle is this being done anywhere else in this php ?

Reply:
I haven't reviewed this module yet but I figured while we're adding new code we can do it correctly here at least.